eventually/vote/models.py

```diff
@@ -231,28 +279,31 @@ def get_by_id(answer_id):
             pass
 
     @staticmethod
-    def create(my_vote=None, my_text=''):
+    def create(members, vote, text=''):
         """
         Static method that create new Answer object
 
-        :param my_vote: foreign key of vote
-        :type my_vote: IntegerField
+        :param vote: foreign key of vote
```

Comment:
maybe it is better to use python's data types like `int`, `bool` not django's `IntegerField` or `BooleanField`